### PR TITLE
Merge Technologies with About platformOS

### PR DESCRIPTION
**Pull request by diana-lakatos**
First round of reorganization on the About platformOs section:
- merged Technologies into this topic
- updated links to technologies
- updated navigation
- reformatted table to bulleted list for better readability

### CODE REVIEW

File: app/views/pages/developer-guide/about-platform-os.liquid [hunk 1]
```diff
@@ -10,17 +10,41 @@ platformOS is a **model-based application development platform**. Aimed at front
...
-  </tr>
-</table>
+platformOS is:
+
+* **Decoupled**: Use frontend code to build custom workflows, B2B, and P2P solutions that interact with any 3rd party API.
+* **Scalable**: Build on load balanced SSL server infrastructure running worldwide on <a href="https://docs.aws.amazon.com" target="_blank" rel="noopener external">AWS</a> (Amazon Web Services) servers.
+* **Secure**: Use our Secure Data Centers with added layers of security, and cater to country-specific security or legal requirements.
+* **Fast**: Use Liquid templating and natively <a href="https://amp.dev" target="_blank" rel="noopener external">AMP</a>-ready GraphQL data calls on a platform tuned for performance, regardless of your application's size.
+* **Extensible**: Integrate your sites with virtually any 3rd party system that provides API access, or build your own APIs in platformOS.
+* **SEO-friendly**: Build on <a href="https://aws.amazon.com/elasticsearch-service" target="_blank" rel="noopener external">AWS Elastic Search</a> and maximize the benefits of local search using our custom model.
```
Comment (pavelloz) [inline app/views/pages/developer-guide/about-platform-os.liquid:20]: @Slashek should we link to aws implementation, or general elastic.io ?

File: app/views/pages/developer-guide/about-platform-os.liquid [hunk 2]
```diff
@@ -10,17 +10,41 @@ platformOS is a **model-based application development platform**. Aimed at front
...
+To help you quickly develop GraphQl queries, we provide a [GraphiQL](https://github.com/graphql/graphiql) graphical user interface through the pos-cli. GraphiQL is a GraphQL editor with autocomplete and live docs that helps you structure GraphQL queries correctly.
+
+<img src="{{ 'images/best-practices/qa/graphiql.png' | asset_url }}" alt="Screenshot of the graphiql editor" />
+
+{% include 'alert/tutorial', content: 'To learn more about using GraphQL and the GraphiQL tool, explore [our documentation topics on using GraphQL in platformOS](/developer-guide/graphql/developing-graphql-queries-using-graphiql).
+' %}
+
+### Node.js
+
+An asynchronous event-driven JavaScript runtime we used to support a small footprint deployment process. Visit the [Official Node.js Documentation](https://nodejs.org/en/docs/) to learn more about Node.js, and [get Node.JS here](https://www.npmjs.com/get-npm).
```
Comment (pavelloz) [inline app/views/pages/developer-guide/about-platform-os.liquid]:
```suggestion
An asynchronous event-driven JavaScript runtime we used to support a small footprint deployment process. Visit the [Official Node.js Documentation](https://nodejs.org/en/docs/) to learn more about Node.js, and [get Node.JS here](https://nodejs.org/en/).
```

File: app/views/pages/developer-guide/about-platform-os.liquid [hunk 3]
```diff
@@ -10,17 +10,41 @@ platformOS is a **model-based application development platform**. Aimed at front
...
+
+{% include 'alert/tutorial', content: 'To learn more about Liquid and how to use it in platformOS, check out our [Liquid Documentation](/api-reference/liquid/introduction) that includes an introduction for beginners, types, tags, filters, whitespace control, and detailed descriptions of platformOS-specific filters and tags — all of them demonstrated with examples.' %}
+
+### GraphQL
+
+A query language used to communicate with our data storages. To learn more, check out the [Official GraphQL Documentation](https://graphql.org/learn/).
+
+Check out platformOS code examples for GraphQL terms in our GraphQl glossary, and see detailed descriptions of queries, mutations, objects, scalars, interfaces, enums, and input objects in our [GraphQL Documenation](/api-reference/graphql/glossary).
+
+To help you quickly develop GraphQl queries, we provide a [GraphiQL](https://github.com/graphql/graphiql) graphical user interface through the pos-cli. GraphiQL is a GraphQL editor with autocomplete and live docs that helps you structure GraphQL queries correctly.
```
Comment (pavelloz) [inline app/views/pages/developer-guide/about-platform-os.liquid:41]: We can link to our GraphiQL tutorial now, its much better than official graphiql docs to be honest ;)

Comment (diana-lakatos) [reply app/views/pages/developer-guide/about-platform-os.liquid:41]: I agree, I linked to it below the image. Do you think I should remove the official GraphiQL link?

Comment (pavelloz) [reply app/views/pages/developer-guide/about-platform-os.liquid:41]: Hmm, i dont know if we should remove official, but our link should be the one people click, if they click only one :)

Comment (diana-lakatos) [reply app/views/pages/developer-guide/about-platform-os.liquid:41]: Thank you! I reorganized the GraphQL section to only include our documentation links (API Reference and Developer Guide topics).

**Review by pavelloz — APPROVED** [review]

**Pull request merged**